packages/styles/scss/components/image/_image.scss

```diff
@@ -37,6 +38,19 @@
     top: calc(50% - 32px);
     right: calc(50% - 32px);
   }
+
+  // dds-image-logo style.
+  :host(#{$dds-prefix}-image-logo) {
```

Comment:
Sorry for the last minute, an integration test is failing and seems to be coming from here:

![image](https://user-images.githubusercontent.com/1641214/142939298-277c8db3-0378-4746-8fd5-bd9e0a9d963d.png)

Reply:
@jeffchew I just added the missing `dds` prefix variable (see - https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/7647/commits/c8ccf4150b98dc0a40b8afe0f29abb1118c8a2b3). Hopefully that should fix the build error.